packages/connection/tests/php/test_Manager_unit.php

```diff
@@ -99,13 +97,9 @@ public function tearDown() {
 	 * @covers Automattic\Jetpack\Connection\Manager::is_active
 	 */
 	public function test_is_active_when_connected() {
```

Comment:
I get 2 failures when running this locally:
```
1) Automattic\Jetpack\Connection\ManagerTest::test_is_active_when_connected
Failed asserting that false is true.

2) Automattic\Jetpack\Connection\ManagerTest::test_is_active_when_not_connected
Expectation failed for method name is equal to 'get_connection_owner_id' when invoked 1 time(s).
Method was expected to be called 1 times, actually called 0 times.
```

tbh I didn't get why we replaced `get_access_token` with `get_connection_owner_id` in this unit test, given that `is_active` still calls `get_access_token`.

Reply:
Ah! Sorry, I have to fix this test. I used to call `has_connected_owner` there instead of `get_access_token`, but decided to keep the old behavior because there would be a slight change on the results.

See discussion on the PR description and here: p9dueE-2bt-p2